src/CoreWCF.Primitives/src/CoreWCF/Description/MessageDescription.cs

```diff
@@ -94,7 +117,7 @@ internal bool IsUntypedMessage
             }
         }
 
-        internal bool IsVoid
+        public bool IsVoid
         {
```

Comment:
Instead of making this public, could the same functionality be implemented in an extension method? MessageType and Body are both public so I think you can.

Reply:
Unfortunately IsTypedMessage is internal.

Reply:
This is addressed in f6e98691eed013e0b35fc2f232b542ef732dd566.